Source/LightboxController.swift

```diff
@@ -47,8 +47,12 @@ public class LightboxController: UIViewController {
 
   public private(set) var page = 0 {
     didSet {
+      var result = max(0, page)
+      result = min(images.count - 1, result)
```

Comment:
Why not. But the solution, I was looking for has to be in standard library, as in Math:
```swift
page = constrain(value, range)
```